tests/shadow/test_shadow_class.py

```diff
@@ -101,8 +102,11 @@ def test_pauli_string_expval(self, shadow):
 
     @pytest.mark.parametrize("H", Hs)
     @pytest.mark.parametrize("shadow", shadows)
+    @pytest.mark.usefixtures("use_legacy_and_new_opmath")
```

Comment:
Adding this to make codecov happy.

Reply:
Before this PR we have legacy logic acting on `LinearCombination`, now that `LinearCombination` is routed to new logic, it exposes the fact that the legacy logic was never tested on legacy `Hamiltonian`, thus making codecov complain.